.travis.yml

```diff
@@ -3,7 +3,6 @@ language: python
 matrix:
   include:
     - python: 2.7
```

Comment:
Can cut 2.7 and 3.5 from the travis matrix.

Reply:
fixed by https://github.com/scikit-tda/kepler-mapper/pull/196/commits/32b94265efa4bba992757bb85c4603fa181931d3